2003-07-17  Guido Guenther  <agx@sigxcpu.org>

	* configure.in: pass $CFLAGS to ld --version-script test

Index: configure.in
===================================================================
RCS file: /cvs/glibc/libc/configure.in,v
retrieving revision 1.401
diff -u -r1.401 configure.in
--- configure.in	3 Jul 2003 21:03:56 -0000	1.401
+++ configure.in	16 Jul 2003 18:35:18 -0000
@@ -1051,7 +1051,7 @@
         global: sym;
 } VERS_1;
 EOF
-  if ${CC-cc} -c $ASFLAGS conftest.s 1>&AS_MESSAGE_LOG_FD 2>&AS_MESSAGE_LOG_FD; then
+  if ${CC-cc} -c $ASFLAGS $CFLAGS conftest.s 1>&AS_MESSAGE_LOG_FD 2>&AS_MESSAGE_LOG_FD; then
     if AC_TRY_COMMAND([${CC-cc} $CFLAGS $LDFLAGS -shared
 				-o conftest.so conftest.o
 				-nostartfiles -nostdlib
